book/01-introduction/sections/about-version-control.asc

```diff
@@ -1,23 +1,23 @@
-=== About Version Control
+=== Versiyon Kontrol
 
 (((version control)))
-What is ``version control'', and why should you care?
-Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later.
-For the examples in this book, you will use software source code as the files being version controlled, though in reality you can do this with nearly any type of file on a computer.
+''Versiyon kontrol'' nedir ve neden önemsenmelidir?
```

Comment:
hey @jnavila,

do we really need to use it like ` ``Versiyon kontrol'' ` or do you think it's okay to keep it like `''Versiyon kontrol''`? 

thanks in advance!

Reply:
Orasını yine yanlış yazmışım, hızlıca bir düzeltme geçeyim hemen

Reply:
The string ` ``Versiyon kontrol''` is a special asciidoc formatting system that will render as ` “Versiyon kontrol” ` . Note that the double quotes are not simple straight double quotes but a pair of matching opening quotes **“** and closing quotes **”** which obey correct English typography. If your language does not follow this typography, you can adapt this. For instance, in German, the quoting would be more like „this“ (opening quotes are on the  bottom), and in French like « this ».
Chances are `''Versiyon kontrol''` is not what you want.

Reply:
Perfect! In Turkish language, we either use it like `' ... '` or `" ... "` but never like `'' ... ''`. Since the opening quotes and closing ones are not matching, I thought it has a special meaning or something, but it's good that we can change it according to our language and its rules. 
In this case, I assume the correct string should be like `'Versiyon kontrol'`. 

Thank you again ^_^

Reply:
I've changed  `''Versiyon kontrol'' `  to `'Versiyon kontrol'`